recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
 
 build:
   number: 0
-  skip: True  # [py3k]
+  skip: True  # [osx and py3k]
```

Comment:
Thanks! I will add those suggestion.

The tokens are not mandatory. Maybe we should also fix the conda-smithy message.